### ISSUE TITLE
Druid Flight Form Exploit in Outlands

### ISSUE THREAD

**Issue by SadAlcoholic** [dalaranwow/dalaran-wow#5197]
[//]: # (INFORMATION ABOUT TAGS)
[//]: # (The black and yellow tag is unique. There can only be one label assigned to them - one black and one yellow)
[//]: # (Priority - ... tags are assigned by team members)
[//]: # (Your issue will be reviewed by testers and the developers will fix it in the future.)
[//]: # (Only once the issue got the label [Fixed - On Live Server] you can find the fix on your Gameserver.)
[//]: # (You can increase the relevance of your issue by adding a thumb-up emote http://i.imgur.com/RFouVVi.png)

[//]: # (***************************************************************)
[//]: # (** !!!!!!!!!!!!!DON'T DELETE THIS TEMPLATE ELSE YOUR ISSUE WILL BE CLOSED!!!!!!!!!!!! **)
[//]: # (***************************************************************)

**Current behaviour**: Tell us what happens in detail (Where, When, What, Why, Who)
I attempted to use flight form to leave the Tomb of Lights at its entrance. 
When done right, this exploit will grant the Druid flight, but not actually put the Druid in flight form (although it is much slower than flight form). The Druid will be able to use weapons and cast spells - it's basically noclip. 
It is possible to land on the ground while retaining flight, and it is possible to shapeshift to any other form (except for flight form) while retaining flight. Yes, flying bears are possible.
It is possible to fly indoors.
Flight is removed when attempting to go through the Dark Portal, and when using hearthstone to get to Kalimdor/Eastern Kingdoms. Unsure whether using a hearthstone to Northrend (and from there going to Kalimdor/Eastern Kingdoms is possible - will update you on it soon.
**Expected behaviour**: Tell us what should happen instead
Flight form should either work properly, or not work at all.
**Steps to reproduce the problem**:

1.  Head to the entrance of the Tomb of Lights in Terokkar Forest as a druid.
2.  Stand right at the entrance, positioning yourself on the slope.
3.  Jump, and while in mid jump, activate flight form before you hit the ground. It might take a couple of attempts to get the exploit working, depending on where you stand and when you activate flight form. It should work when you see your character switch to flight form, and then suddenly come out of flight form as you hit the ground. That's all you need to do.

**Include proofs for this behaviour**

You can research this on several pages like http://wowhead.com or http://wowwiki.wikia.com or http://www.youtube.com/
Issues without proofs will be closed and tag On Hold will be added until the proof is provided.

**Include the ID for the game objects, npcs (creatures,pets,minions), spells, items, quests, instances, zones, achievements, skills**

You can research this on several pages like https://wowgaming.altervista.org/aowow/ or https://wowhead.com/

**Include Screenshots from the issue if necessary**
https://imgur.com/a/lKOSG

**Comment by dalwowreportissue** [comment]
This isn't druid or outland specific, reproducible on my dk in wintergrasp:
![wowscrnshot_123017_201331](https://user-images.githubusercontent.com/34975812/34459226-f330fa34-ed9e-11e7-8bc2-73f42714ab71.jpg)
That is swimming in the air while able to cast spells.

**Comment by SadAlcoholic** [comment]
Really? I thought this was druid specific because of flight form's instant cast. How were you able to do it with the 1.5 sec casting time?

**Comment by dalwowreportissue** [comment]
I got this to trigger by jumping at the end of the cast.
